skyplane/cli/cli_init.py

```diff
@@ -322,7 +322,7 @@ def disable_gcp_support():
 
         # check if GCP is enabled
         inferred_cred, inferred_project = compute.GCPAuthentication.get_adc_credential()
-        if inferred_cred is None or inferred_project is None:
+        if inferred_cred is None:  # or inferred_project is None:
```

Comment:
is this since we will ask for the project if not set?

Reply:
You can be authenticated with GCP but not have a default project set, so instead of disabling GCP we should use the user provided project ID if `inferred_project` is None